lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,27 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
+    <%- edx.StringUtils.interpolate(
           gettext("You still need to visit the %(display_name)s website to complete the credit process."),
           { display_name: display_name }, true
         ) %>
   </div>
   <div class="provider-more-info">
-    <%= interpolate(
+    <%- edx.StringUtils.interpolate(
           gettext("To finalize course credit, %(display_name)s requires %(platform_name)s learners to submit a credit request."),
           { display_name: display_name, platform_name: platformName }, true
         ) %>
   </div>
   <div class="provider-instructions">
-    <%= fulfillment_instructions %>
+    <%- fulfillment_instructions %>
   </div>
 </div>
 
 <div class="provider-buttons-logos">
   <div class="provider-logo">
-    <%= interpolate("<img src='%s' alt='%s'></image>", [thumbnail_url, display_name]) %>
+    <%- edx.StringUtils.interpolate("<img src='%s' alt='%s'></image>", [thumbnail_url, display_name]) %>
   </div>
   <div class="complete-order">
-    <%= interpolate('<button data-provider="%s" data-course-key="%s" data-username="%s" class="complete-course" onClick=completeOrder(this)>%s</button>', [id, course_key, username,
+    <%- edx.StringUtils.interpolate('<button data-provider="%s" data-course-key="%s" data-username="%s" class="complete-course" onClick=completeOrder(this)>%s</button>', [id, course_key, username,
         gettext( "Get Credit")]) %>
```

Comment:
So here you are using fully qualified name for StringUtils. It seems there are two variations used. 1) This way 2) Without prefixing with edx like https://github.com/edx/edx-platform/blob/master/lms/templates/learner_dashboard/course_card.underscore#L29. So please do test this that you have this function available here.